src/Core/Content/Seo/SeoUrlGenerator.php

```diff
@@ -31,7 +32,11 @@ class SeoUrlGenerator
 {
     final public const ESCAPE_SLUGIFY = 'slugifyurlencode';
 
+    /* private readonly TwigVariableParser $twigVariableParser; */
+
     /**
+     * @deprecated tag:v6.6.0 the $twigVariableParser parameter will be removed use $parserFactory instead
```

Comment:
the constructor of this service is marked as internal, therefore you can already change it right now, it is no Breaking Change for us as you should always get the services from the container and not create them manually